src/main/java/org/killbill/billing/plugin/stripe/StripeCheckoutServlet.java

```diff
@@ -59,10 +59,21 @@ public StripeCheckoutServlet(final OSGIKillbillClock clock,
 
     @POST
     public Result createSession(@Named("kbAccountId") final UUID kbAccountId,
-                                @Named("successUrl") final Optional<String> successUrl,
+					            @Named("successUrl") final Optional<String> successUrl,
+					            @Named("cancelUrl") final Optional<String> cancelUrl,
+					            @Named("name") final Optional<String> line_item_name,
+					            @Named("kbaccount") final Optional<String> kbaccount,
+					            @Named("kbinvoice") final Optional<String> kbinvoice,
```

Comment:
Let's use `kbInvoiceId` for both the query parameter and the variable, for consistency.